src/osp/Active/drawing.h

```diff
@@ -80,16 +73,62 @@ using TexIdOwner_t      = TexRefCount_t::Owner_t;
  */
 struct ACtxDrawing
 {
+    // Scene-space Meshes
+    lgrn::IdRegistryStl<MeshId>             m_meshIds;
+    MeshRefCount_t                          m_meshRefCounts;
+
+    // Scene-space Textures
+    lgrn::IdRegistryStl<TexId>              m_texIds;
+    TexRefCount_t                           m_texRefCounts;
+};
+
+/**
+ * @brief Associates mesh/texture resources Ids from ACtxDrawing with Resources
+ */
+struct ACtxDrawingRes
+{
+    // Required for std::is_copy_assignable to work properly inside of entt::any
+    ACtxDrawingRes() = default;
+    ACtxDrawingRes(ACtxDrawingRes const& copy) = delete;
+    ACtxDrawingRes(ACtxDrawingRes&& move) = default;
+
```

Comment:
not also defining operator= ?

Reply:
Noted. At the moment this is harmless as no assignments are done.

I'm planning on reorganizing and cleaning the code on the next PR, as functions are a little scattered after all the rewriting and it's hard to navigate. I'll probably add a "move-only" macro for this.